Modules/+Sources/@VelocityLaser/VelocityLaser.m

```diff
@@ -211,6 +229,40 @@ function delete(obj)
                 rethrow(err)
             end
         end
+        
+        function val = set_wheel_ip(obj,val,~)
+            err = obj.connect_driver('wheel', 'ArduinoServo', val, obj.wheel_pin);
+            if isempty(obj.wheel)
+                if ~isempty(err)
+                    rethrow(err)
+                end
```

Comment:
It seems this block and line 242 below could be consolidated outside this if block?